spec/features/providers_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature 'View providers', type: :feature do
+  scenario 'Navigate to /organisations' do
+    stub_omniauth
+    stub_session_create
+    stub_backend_api
```

Comment:
A little bit tied down, but I guess when we work more on it it will evolve better

Reply:
Exactly @defong. Going to catch up with @misaka today on a testing strategy for this app but this is fine for now.